src/components/ProfileCard/ProfileCard.docs.mdx

```diff
@@ -0,0 +1,379 @@
+# ProfileCard
```

Comment:
The entire documentation file is written in Spanish. Project rules require English as the standard language for code, comments and docs.

**Sugerencia:**
```typescript
Translate the MDX documentation to English (title, descriptions, examples, notes). Keep one language consistently across the repository. If Spanish docs are required, add an English version too and mark localized docs appropriately.
```